Dockerfile

```diff
@@ -44,4 +45,4 @@ RUN mix do compile, release
 EXPOSE 5000
 
 # The command to run when this image starts up
-CMD ["_build/prod/rel/slax/bin/slax", "foreground"]
+# CMD ["_build/prod/rel/slax/bin/slax", "start"]
```

Comment:
We're going to have to pull in the latest `master` and update the `heroku.yml` file with the correct command as well